docs-site/content/guide/tips-for-filtering.md

```diff
@@ -0,0 +1,253 @@
+- Michael Johansson
+
+When you use the prefix in conjunction with the exact-match operator `:=`, the results will only include entries where the prefix matches the beginning of the name:
+
+```
+name:=Jo*
+```
+
+This will match:
+
+- ✅ Jonathan Jacobs
+- ✅ John McKinley
+
+But not:
+
+- ❌ Michael Johansson
+
+## Boolean Operations
+
+You can combine multiple filters using logical operators.
+
+| Operator                  | Description | Example                                                |
+| ------------------------- | ----------- | ------------------------------------------------------ |
+| `&&`                      | Logical AND | `country:=USA && city:=New York`                       |
+| <code>&#124;&#124;</code> | Logical OR  | <code>country:=USA &#124;&#124; country:=Canada</code> |
+
+When using logical operators, you can use parentheses to group your filters.
+
+For example, to filter for products that are either made in the USA or Canada, and are located in New York, you can use the following filter:
+
+```
+(country:=USA || country:=Canada) && city:=New York
+```
+
+This will return products that are either made in the USA or Canada, and are located in New York.
+When using logical operators, you can use parentheses to group your filters.
+For example, to filter for products that are either made in the USA or Canada, and are located in New York, you can use the following filter:
+
+```
+(country:=USA || country:=Canada) && city:=New York
+```
+
+### Operator Precedence
+
+In boolean operations, the AND operator (`&&`) has higher precedence than the OR operator (`||`). This means that AND operations are performed before OR operations, unless parentheses are used to specify a different order.
+
+For example:
+
+```
+country:=USA || country:=Canada && city:=New York
+```
+
+Would be interpreted as:
+
+```
+country:=USA || (country:=Canada && city:=New York)
+```
+
+To change the order of operations, use parentheses:
+
+```
+(country:=USA || country:=Canada) && city:=New York
+```
+
+## Filtering Array Types
+
+Any of the array types can be used for filtering, matching any of their values.
+
+For example, if there's a field called `department_prices` that's a `int32[]` and there's a document with `department_prices: [32, 60, 80]`,
+then the filter of:
+
+```
+department_prices:<80
+```
+
+Would in turn return this document.
+
+Subsequently, you can also use array operators:
+
+```
+deparment_prices:[20...80]
+```
+
+## Escaping special characters
+
+If you want to filter for a value that includes special characters, such as a comma (`,`), you can use
+backticks around your filter value:
+
+```
+country:=`United States, Minor Outlying Islands`
+```
+
+## Filtering Geopoints
+
+For information on filtering `geopoint` fields to search within a specific area, please refer to our <RouterLink :to="`/${$site.themeConfig.typesenseLatestVersion}/api/geosearch.html#searching-within-a-radius`">documentation</RouterLink> on Geosearch.
```

Comment:
@tharropoulos Could you add actual examples here of geo filtering, just like in the JOINs section, and then link to the geo search docs?

Reply:
> @tharropoulos Could you add actual examples here of geo filtering, just like in the JOINs section, and then link to the geo search docs?

Addressed in 29f67a3ac78f3e7a3b18b54029c6d299fac9de4f